README.rst

```diff
@@ -184,15 +184,14 @@ Answer the prompts with your own desired options_. For example::
     5 - 9.5
     Choose from 1, 2, 3, 4, 5 [1]: 1
     Select js_task_runner:
-    1 - None
-    2 - Gulp
+    1 - Webpack
+    2 - None
```

Comment:
```suggestion
    1 - None
    2 - Webpack
```
The default should be None as not everybody has or wants a js_task_runner (additionally, None was the original default).